app/views/comments/index.html.erb

```diff
@@ -0,0 +1,72 @@
+<div class="container-fluid text-center">
+  <br>
+  <div class="row">
+    <div class="col-2"></div>
+    <div class="col-8">
+
+      <table class="table table-hover table-bordered">
+        <thead>
+          <tr>
+            <th>
+              <div class="row">
+                <div class="col-10 h4 text-left">
+                  <%= @event.name %>
+                </div>
+                <div class="col-2">
+                  <button type="button" class="btn btn-sm btn-info" data-toggle="modal" data-target="#addcomment">
+                    Add comment
+                  </button>
+                </div>
+              </div>
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          <% @comments.each do |comment| %>
+            <tr>
+              <td class="text-left">
+                  <strong>User:</strong> <%= comment.user.username.titleize %>
+                  <br>
+                  <strong>At:</strong> <%= comment.created_at %>
+                  <br>
+                  <strong>Comment:</strong> <%= comment.comment %>
+                  <br>
+                  <% if comment.likes.pluck(:user_id).include?(session[:current_user_id]) %>
```

Comment:
Don't fire the query on the view page, use a controller for it.
and you can change it to something like this
Like.find_by(comment_id: comment.id, user_id: session[:current_user_id]).present?